[1m============================= test session starts ==============================[0m
platform linux -- Python 3.10.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.0
cachedir: /workspaces/codespace/ros2/project/tutorial/build/launch_testing_examples/.pytest_cache
rootdir: /workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples
plugins: colcon-core-0.15.2
[1mcollecting ... [0m[1m
collected 0 items / 9 errors                                                   [0m

==================================== ERRORS ====================================
[31m[1m______ ERROR collecting launch_testing_examples/check_msgs_launch_test.py ______[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/launch_testing_examples/check_msgs_launch_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
launch_testing_examples/check_msgs_launch_test.py:17: in <module>
    import launch
E   ModuleNotFoundError: No module named 'launch'[0m
[31m[1m_ ERROR collecting launch_testing_examples/check_multiple_nodes_launch_test.py _[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/launch_testing_examples/check_multiple_nodes_launch_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
launch_testing_examples/check_multiple_nodes_launch_test.py:21: in <module>
    import launch
E   ModuleNotFoundError: No module named 'launch'[0m
[31m[1m______ ERROR collecting launch_testing_examples/check_node_launch_test.py ______[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/launch_testing_examples/check_node_launch_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
launch_testing_examples/check_node_launch_test.py:18: in <module>
    import launch
E   ModuleNotFoundError: No module named 'launch'[0m
[31m[1m_____ ERROR collecting launch_testing_examples/hello_world_launch_test.py ______[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/launch_testing_examples/hello_world_launch_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
launch_testing_examples/hello_world_launch_test.py:17: in <module>
    import launch
E   ModuleNotFoundError: No module named 'launch'[0m
[31m[1m____ ERROR collecting launch_testing_examples/record_rosbag_launch_test.py _____[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/launch_testing_examples/record_rosbag_launch_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
launch_testing_examples/record_rosbag_launch_test.py:22: in <module>
    import launch
E   ModuleNotFoundError: No module named 'launch'[0m
[31m[1m______ ERROR collecting launch_testing_examples/set_param_launch_test.py _______[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/launch_testing_examples/set_param_launch_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
launch_testing_examples/set_param_launch_test.py:18: in <module>
    import launch
E   ModuleNotFoundError: No module named 'launch'[0m
[31m[1m___________________ ERROR collecting test/test_copyright.py ____________________[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/test/test_copyright.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test/test_copyright.py:15: in <module>
    from ament_copyright.main import main
E   ModuleNotFoundError: No module named 'ament_copyright'[0m
[31m[1m_____________________ ERROR collecting test/test_flake8.py _____________________[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/test/test_flake8.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test/test_flake8.py:15: in <module>
    from ament_flake8.main import main_with_errors
E   ModuleNotFoundError: No module named 'ament_flake8'[0m
[31m[1m_____________________ ERROR collecting test/test_pep257.py _____________________[0m
[31mImportError while importing test module '/workspaces/codespace/ros2/project/tutorial/src/examples/launch_testing/launch_testing_examples/test/test_pep257.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test/test_pep257.py:15: in <module>
    from ament_pep257.main import main
E   ModuleNotFoundError: No module named 'ament_pep257'[0m
- generated xml file: /workspaces/codespace/ros2/project/tutorial/build/launch_testing_examples/pytest.xml -
=========================== short test summary info ============================
ERROR launch_testing_examples/check_msgs_launch_test.py
ERROR launch_testing_examples/check_multiple_nodes_launch_test.py
ERROR launch_testing_examples/check_node_launch_test.py
ERROR launch_testing_examples/hello_world_launch_test.py
ERROR launch_testing_examples/record_rosbag_launch_test.py
ERROR launch_testing_examples/set_param_launch_test.py
ERROR test/test_copyright.py
ERROR test/test_flake8.py
ERROR test/test_pep257.py
!!!!!!!!!!!!!!!!!!! Interrupted: 9 errors during collection !!!!!!!!!!!!!!!!!!!!
[31m============================== [31m[1m9 errors[0m[31m in 0.08s[0m[31m ===============================[0m
